src/components/InputCopy/index.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Button from '../../components/Button';
+import Input from '../../components/Input';
+import './InputCopy.scss';
+
+export default class InputCopy extends React.Component {
+  state = {
+    alert: false
+  };
+
+  render() {
+    return (
+      <div className="input-copy">
+        {this.state.alert && (
+          <div className="input-copy__alert">
+            <i className="ion-checkmark-circled" /> Copied to clipboard.
+          </div>
+        )}
+        <div>
+          <Button
+            text="Copy"
+            className="input-copy__btn"
```

Comment:
From the screenshot it looks like a standard primary button, so you shouldn't have to pass in a custom `className`?

Reply:
hmm that class is used to position the button to the right. Do you think it's better to add an additional element containing the button and apply the class to that one instead?

Reply:
hmm what about just doing `display: flex` on the parent?

Reply:
haha great idea! I have to start using `flex` more